eslint.config.mjs

```diff
@@ -46,7 +46,7 @@ export default [
       ecmaVersion: "latest",
       sourceType: "script",
       parserOptions: {
-        project: true,
+        projectService: true,
```

Comment:
The same issue as https://github.com/ota-meshi/eslint-plugin-yml/pull/346#discussion_r1667218685! ❗ 

```plaintext

/Users/josh/repos/eslint-plugin-jsonc/docs/.vitepress/theme/components/components/EslintPluginEditor.vue
  0:0  error  Parsing error: "parserOptions.programs" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided program instance(s): docs/.vitepress/theme/components/components/EslintPluginEditor.vue

/Users/josh/repos/eslint-plugin-jsonc/docs/.vitepress/theme/components/components/PgEditor.vue
  0:0  error  Parsing error: "parserOptions.programs" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided program instance(s): docs/.vitepress/theme/components/components/PgEditor.vue

/Users/josh/repos/eslint-plugin-jsonc/docs/.vitepress/theme/components/components/RulesSettings.vue
  0:0  error  Parsing error: "parserOptions.programs" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided program instance(s): docs/.vitepress/theme/components/components/RulesSettings.vue

/Users/josh/repos/eslint-plugin-jsonc/docs/.vitepress/theme/components/components/SnsBar.vue
  0:0  error  Parsing error: "parserOptions.programs" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided program instance(s): docs/.vitepress/theme/components/components/SnsBar.vue

/Users/josh/repos/eslint-plugin-jsonc/docs/.vitepress/theme/components/eslint-code-block.vue
  0:0  error  Parsing error: "parserOptions.programs" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided program instance(s): docs/.vitepress/theme/components/eslint-code-block.vue

/Users/josh/repos/eslint-plugin-jsonc/docs/.vitepress/theme/components/playground-block.vue
  0:0  error  Parsing error: "parserOptions.programs" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided program instance(s): docs/.vitepress/theme/components/playground-block.vue
```

Reply:
As with that issue, this should be fixed as https://github.com/typescript-eslint/typescript-eslint/pull/9580.